src/screens/AirdropDashboard/components/AirdropInProgress.tsx

```diff
@@ -0,0 +1,129 @@
+import { CompositeNavigationProp } from '@react-navigation/native';
+import { StackNavigationProp } from '@react-navigation/stack';
+import React, { FC, RefObject } from 'react';
+import { View, StyleSheet, Text } from 'react-native';
+import Carousel from 'react-native-snap-carousel';
+
+import { images } from 'app/assets';
+import { Image, Countdown, AirdropWalletsList, AirdropWalletsCarousel } from 'app/components';
+import { CONST, Route, RootStackParams, AirdropWalletDetails } from 'app/consts';
+import { getFormattedAirdropDate } from 'app/helpers/airdrop';
+import { typography, palette } from 'app/styles';
+
+import { Footer, AvailableWalletAction, RegisteredWalletAction } from './';
+
+const i18n = require('../../../../loc');
+
+interface Props {
+  navigation: CompositeNavigationProp<
+    StackNavigationProp<RootStackParams, Route.MainTabStackNavigator>,
+    StackNavigationProp<RootStackParams, Route.AirdropDashboard>
+  >;
+}
+
+// TODO: ultimately should come from API / Redux store
```

Comment:
you can start mocking the API responses and implement the whole logic, create airdropApi.ts here and mock the endpoints
![Zrzut ekranu 2021-01-25 o 09 35 01](https://user-images.githubusercontent.com/67273035/105680854-a34fa580-5ef0-11eb-9d9a-dd5da7f4d439.png)